packages/web/src/ui/views/examples.js

```diff
@@ -1,75 +1,49 @@
-  { file: 'parameters/stepper-motor.jscad', title: 'Interactive Params: Stepper Motor' },
-  { file: 'parameters/iphone4-case.jscad', title: 'Interactive Params: iPhone4 Case' },
-  { file: 'parameters/name_plate.jscad', title: 'Interactive Params: Name Plate' },
-  { file: 'parameters/balloons.jscad', title: 'Interactive Params: Balloons' },
-
-  { file: 'globe.jscad', title: 'Include(): Globe', spacing: true },
-  { file: 'platonics/', title: 'Recursive Include(): Platonics' },
-
-  { file: 'babypanda2.svg', title: 'SVG Image: Baby Panda', spacing: true },
-
-  { file: '3d_sculpture-VernonBussler.stl', title: '3D Model: 3D Sculpture (Vernon Bussler)', type: 'STL', spacing: true },
-  { file: 'frog-OwenCollins.stl', title: '3D Model: Frog (Owen Collins)', type: 'STL' },
-  { file: 'thing_7-Zomboe.stl', title: '3D Model: Thing 7 / Flower (Zomboe)', type: 'STL' },
-  { file: 'yoda-RichRap.stl', title: '3D Model: Yoda (RichRap)', type: 'STL' },
-  { file: 'feathers_mcgraw.stl', title: '3D Model: Feathers Mcgraw (q1g0ng)', type: 'STL' }
+  { file: 'core/primitives/primitives2D.js', title: '2D Primitives' },
+  { file: 'core/primitives/primitives3D.js', title: '3D Primitives' },
+  { file: 'core/primitives/polyhedron.js', title: 'Custom Polyhedron' },
+  { file: 'core/primitives/torus.js', title: 'Torus' },
+  { file: 'core/primitives/dodecahedron.js', title: 'Dodecahedron' },
+  { file: 'core/primitives/roundedCuboid.js', title: 'Rounded Cuboid' },
+  { file: 'core/primitives/sphere.js', title: 'Sphere Variations' },
+
+  { file: 'core/booleans/basicBooleans.js', title: 'Basic Booleans' },
+  { file: 'core/extrusions/basicExtrusions.js', title: 'Basic Extrusions' },
+  { file: 'core/extrusions/extrudeFromSlices.js', title: 'Extrude From Slices' },
+  { file: 'core/hulls/hull.js', title: 'Hull & Hull Chain' },
+  { file: 'core/text/text.js', title: 'Creating Text' },
+  { file: 'core/expansions/offset.js', title: 'Offset Geometry' },
+  { file: 'core/expansions/expand.js', title: 'Expand Path or Object' },
+  { file: 'core/transforms/center.js', title: 'Center Geometries' },
+
+  { file: 'core/colors/basicColors.js', title: 'Basic Colors' },
+  { file: 'core/colors/colorCube.js', title: 'Color Cube' },
+  { file: 'core/colors/transparency.js', title: 'Transparency' },
+
+  { file: 'core/measurements/measureBounds.js', title: 'Measure Bounds' },
+  { file: 'core/measurements/measureAreaAndVolume.js', title: 'Measure Area & Volume' },
+  { file: 'core/curves/bezier/simpleExtrude.js', title: 'Simple Extrude' },
+  { file: 'core/curves/bezier/extrudeAlongPath.js', title: 'Extrude Along a Path' },
+
+  { file: 'parameters/allParamTypes.js', title: 'All Parameter Types' },
+  { file: 'parameters/gear.js', title: 'Involute Gear' },
+  { file: 'parameters/balloons.js', title: 'Birthday Balloons' },
+
+  { file: 'module-design/', title: 'Modular Design' },
+
+  { file: 'import/SVGImport/', title: 'SVG Import' },
```

Comment:
Loading folders does not work in the UI.  Would suggest a separate PR for that.

Reply:
Just to think further... why can't this be generated by/from the examples project?

Reply:
> Loading folders does not work in the UI. Would suggest a separate PR for that.

Are you going to take this next? if not then let's create a new issue to track.

Reply:
> Just to think further... why can't this be generated by/from the examples project?
> 
> it seems that this information is static and redundant. and possibly incorrect.

If the file annotations are good, it probably should be. One thing that I felt was important was to  order the examples in terms of complexity, and I want to soon group them by topic.  I'll add an issue for this. https://github.com/jscad/OpenJSCAD.org/issues/673

Reply:
> Are you going to take this next? if not then let's create a new issue to track.

I might, but I have a bunch to learn first, so take it if you want.  https://github.com/jscad/OpenJSCAD.org/issues/674